lib/components/GalleryCard/GalleryCard.module.scss

```diff
@@ -106,7 +99,15 @@ $card-fixed-content-height: $gallery-card-height - $gallery-card-background-heig
 
 .background-color {
   @extend %card-background;
-  background-color: var(--color-bg-gallery-tile-default);
+  background-color: var(--color-content-background-primary);
+
+  &.blue {

```

Comment:
what blue means?

Reply:
it was there before, I didn't delete it for backwards compat... it used to mean you are hardcoding the gallery card to be blue. I also don't like it haha